server/package.json

```diff
@@ -9,8 +9,8 @@
   "main": "server.js",
   "scripts": {
     "prebuild": "tsoa spec-and-routes",
-    "build": "tsc",
-    "dev": "concurrently \"nodemon\" \"nodemon -x tsoa spec-and-routes\"",
+    "build": "yarn prebuild && tsc",
```

Comment:
is it useful? It seems that prebuild is called automatically before the build. In this case, `tsoa spec-and-routes` is runned twice.
But for the `dev` script I think we should keep it, because the `tsoa` script is not complete before `tsc` runs, which causes an error

Reply:
Hmm yes I noticed that with `yarn` v1, the prebuild script isn't called at all when running `yarn build.`. That's why I called it explicitly in the updated build script. I think the prebuild name is misleading though.  On the other hand, when the dist directory is missing, running `yarn dev:server` was failing because of a race condition between the server and `tsoa spec-and-routes`.  That was the reason for my update here.